nncf/common/graph/graph.py

```diff
@@ -178,6 +178,19 @@ def get_previous_nodes(self, node: NNCFNode) -> List[NNCFNode]:
         nx_node_keys = self._nx_graph.pred[self._node_id_to_key_dict[node.node_id]]
         return [self._nx_node_to_nncf_node(self._nx_graph.nodes[key]) for key in nx_node_keys]
 
+    def get_previous_nodes_sorted_by_in_port(self, node: NNCFNode) -> List[NNCFNode]:
```

Comment:
What reason do this function is added to the graph class? Can you sort nodes by in_port in the algorithm?

Reply:
@alexsu52 I think that this is a convenience function which may stay in the graph itself for the time being, since it introduces no new dependencies. May refactor that later; meanwhile the fix here is critical for the release, so I'll merge this - hope you don't mind.

Reply:
@vshampor Could you open a ticket for this issue?

Reply:
Ref. 53414 opened.